main.go

```diff
@@ -1,69 +1,59 @@
-// A minimal example of how to include Prometheus instrumentation.
 package main
```

Comment:
a general comment:
I personally would split this up into a number of files, they still can all be in the same package `main` but typically a main.go file will contain intiialiazation and endpoints not logic and functions. Basically the `main` at the end is almost the only thing I would have here, maybe connections as well (although that also can be extracted out). You can have a `Types.go` to hold your structs as well.

Reply:
That's my style in Java also. The class with `public static void main(String[] arg)` has nothing else but helpers for main().

(Personally, I would have made starting an app invoking a constructor, maybe call `Application(String[] av, Context c)`, but that's a bit off course for a discussion of Go code. Even for me.

Reply:
Split it out in e67cb371a64fd6b3b34f5a82fe2476b139227947.